packages/orchestration/src/ethereum-api.ts

```diff
@@ -2,6 +2,8 @@
  * Info for an Ethereum-based chain.
  */
 export type EthChainInfo = Readonly<{
+  // XXX consider ~BaseChainInfo type, with `cctpDestinationDomain` + `chainId`
+  cctpDestinationDomain?: number;
   chainId: string;
```

Comment:
@0xpatrickdev how does this relate to the meeting we just had?
I don't see much in the meeting notes.